config/locales/en.yml

```diff
@@ -151,6 +151,31 @@ en:
     numerator: "Number of days until a patient's next scheduled appointment"
     denominator: "Total appointments scheduled for follow-up patients during a month."
 
+  overdue_patients_definition: "A patient with a scheduled visit date which has passed with no visit."
+  overdue_patients:
+    reports_card_subtitle: "Hypertension patients that are overdue at the start of the month"
+    numerator: "Patients that are overdue at the start of the month."
+    denominator: "Hypertension patients assigned to %{region_name}. Deleted, dead and lost to follow-up patients are excluded."
+  overdue_patients_called: 
+      reports_card_subtitle: "Total overdue hypertension patients called during the month."
+      numerator: "Unique patients with at least one ‘Result of call...’ marked during the month."
+      numerator_note: "Note: This number includes patients that become overdue during the month and received a call, they may not be part of the denominator."
+      denominator: "Hypertension patients assigned to %{region_name} that are overdue at the start of the month. Deleted, dead and lost to follow-up are excluded."
+  overdue_patients_call_activity: 
+    reports_card_subtitle: "The number of unique overdue hypertension patients that were called during the month."
+    patients_called: "Unique patients with at least one ‘Result of call...’ marked during the month."
+    percent_called_numerator: "Unique patients with at least one ‘Result of call...’ marked by user during the month."
+    percent_called_denominator: "Hypertension patients assigned to %{region_name} that are overdue at the start of the month. Deleted, dead and lost to follow-up are excluded."
+  overdue_toggle_tooltip:
+    main: "Patients marked ‘Removed from overdue list’ before the 1st of the month and patients without a phone number are excluded."
+    additional: |
```

Comment:
The strings show up with quotes in the tooltip.

<img width="210" alt="Screenshot 2023-06-08 at 1 34 34 PM" src="https://github.com/simpledotorg/simple-server/assets/3933133/42743b17-ab42-4ac2-a4c7-238e508dedbf">